src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1519,4 +1519,19 @@
     <value>and all other platforms</value>
     <comment>This call site is reachable on: 'windows' 10.0.2000 and later, and all other platforms</comment>
   </data>
+  <data name="ProvidePublicParameterlessSafeHandleConstructorDescription" xml:space="preserve">
+    <value>Providing a public parameterless constructor for a type derived from 'System.Runtime.InteropServices.SafeHandle' enables better performance and usage with source-generated interop solutions.</value>
+  </data>
+  <data name="ProvidePublicParameterlessSafeHandleConstructorMessage" xml:space="preserve">
+    <value>Provide a public parameterless constructor for the '{0}' type that is derived from 'System.Runtime.InteropServices.SafeHandle'</value>
+  </data>
+  <data name="ProvidePublicParameterlessSafeHandleConstructorTitle" xml:space="preserve">
+    <value>Provide a public parameterless constructor for concrete types derived from 'System.Runtime.InteropServices.SafeHandle'</value>
+  </data>
+  <data name="AddPublicParameterlessConstructor" xml:space="preserve">
+    <value>Add a public constructor that takes zero parameters.</value>
+  </data>
+  <data name="MakeParameterlessConstructorPublic" xml:space="preserve">
+    <value>Make the constructor that takes zero parameters 'public'.</value>
+  </data>
```

Comment:
I'm not sure if codefix titles should end in a period or not. There is inconsistency about this in the existing code fix titles:

https://github.com/dotnet/roslyn-analyzers/blob/8a7a50734c7f2e983b4075a86ca825e689e7833e/src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx#L1362-L1367

Tagging @mavasani for the preferred option.

Reply:
Agreed, both diagnostic and code fix title should not end with a period.